OWNERS

```diff
@@ -11,6 +13,7 @@ approvers:
 reviewers:
   - ashley-cui
   - baude
+  - rhatdan 
```

Comment:
The tools don't care, but if we're keeping this list alphabetically sorted, this needs to be tweaked.  Not a blocker either way.